tasks/components/layout/TaskDetailView.tsx

```diff
@@ -0,0 +1,159 @@
+          <span className={tw('text-black')}>{translation.close}</span>
+          <X />
+        </button>
+      </div>
+      <div className={tw('flex flex-row flex-1 gap-x-8 mt-3')}>
+        <div className={tw('flex flex-col gap-y-4 min-w-[500px]')}>
+          <div className={tw('min-h-1/4')}>
+            <Textarea
+              headline={translation.notes}
+              value={task.description}
+              onChange={description => onChange({ ...task, description })}
+            />
+          </div>
+          <SubtaskView subtasks={task.subtasks} onChange={subtasks => onChange({ ...task, subtasks })} />
+        </div>
+        <div className={tw('flex flex-col justify-between min-w-[250px]')}>
+          <div className={tw('flex flex-col gap-y-4')}>
+            <div>
+              <label>{translation.assignee}</label>
+              <Select
+                value={task.assignee}
+                options={[
+                  { label: 'Assignee 1', value: 'assignee1' },
+                  { label: 'Assignee 2', value: 'assignee2' },
+                  { label: 'Assignee 3', value: 'assignee3' },
+                  { label: 'Assignee 4', value: 'assignee4' }
+                ]}
+                onChange={assignee => onChange({ ...task, assignee })}
+              />
+            </div>
+            <div>
+              <label>{translation.dueDate}</label>
+              {/* TODO use some date picker component */}
+              <Select
+                value={task.dueDate}
+                options={[
+                  { label: 'Finished', value: 'Finished' },
+                  { label: '02.03.2001', value: '02.03.2001' },
+                  { label: '17.07.2011', value: '17.07.2011' },
+                  { label: '01.04.2014', value: '01.04.2014' }
+                ]}
+                onChange={dueDate => onChange({ ...task, dueDate })}
+              />
+            </div>
+            <div>
+              <label>{translation.status}</label>
+              <TaskStatusSelect value={task.status} onChange={status => onChange({ ...task, status })}/>
+            </div>
+            <div>
+              <label>{translation.visibility}</label>
+              <Select
+                value={task.isPublicVisible}
+                options={[
+                  { label: translation.private, value: false },
+                  { label: translation.public, value: true }
+                ]}
+                onChange={isPublicVisible => onChange({ ...task, isPublicVisible })}
+              />
+            </div>
+          </div>
+          {task.creationDate !== undefined ? (
+            <div className={tw('flex flex-col gap-y-8 mt-16')}>
+              <div className={tw('flex flex-col')}>
+                <span className={tw('font-bold text-gray-800 mb-1')}>{translation.creationTime}</span>
+                <span>{task.creationDate.toString()}</span>
```

Comment:
Use format string for better representation.
You're able to use the datetime provided by the os.

Empathize with a potential user... Is it relevant to see the timezone information? The exact second?